apps/judicial-system/backend/migrations/20241022142128-update-defendant.js

```
@@ -0,0 +1,25 @@
+'use strict'
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)